src/main/kotlin/nl/info/zac/app/informatieobjecten/exception/ConvertException.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ *
+ */
+package nl.info.zac.app.informatieobjecten.exception
+
+import nl.info.zac.exception.ErrorCode.ERROR_CODE_CONVERT_NOT_POSSIBLE
+import nl.info.zac.exception.InputValidationFailedException
+
+class ConvertException : InputValidationFailedException(
```

Comment:
Maybe rename it to make it more meaningful? 'convert' does not say much.
```suggestion
class EnkelvoudigInformatieObjectConversionException : InputValidationFailedException(
```

Reply:
Good remark! Will rename this.